get-openssl.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+VSN="${1:-1.1.1n}"
+NAME="openssl-$VSN"
+FILE="$NAME.tar.gz"
+curl -o "$FILE" -f -L "https://www.openssl.org/source/$FILE"
+
+tar zxf "$FILE"
+cd "$NAME"
```

Comment:
Maybe `pushd` + `popd` would be safer than assuming that the previous folder is always the root?